### ISSUE TITLE
Global sprite should have a aria-hidden="true" attribute

### ISSUE THREAD

**Issue by zhell** [vbenjs/vite-plugin-svg-icons#90]
Hi, I really love the plugin, I've been using it on almost all my front-end from scratch projects.
On thing I noticed, is in terms of accessibility, the sprite appended to the page should have an aria-hidden="true" attribute, preventing the screen reader's ability to focus on it, even if it's positioned outside of the screen via css.
